contrib/platform/test/com/sun/jna/platform/win32/Secur32Test.java

```diff
@@ -190,84 +190,6 @@ public void testAcceptSecurityContext() {
                 phClientCredential));
     }
 
-    public void testImpersonateRevertSecurityContext() {
```

Comment:
What's the purpose of moving it out?

Reply:
I noticed, that the test following this method failed, moving the impersonation test out, stabilized this.

Reply:
So it fails to cleanup something :( I would maybe just reorder them and add a // BUGBUG to it or something?